packages/xo-server/src/api/vm.mjs

```diff
@@ -540,6 +540,20 @@ insertCd.resolve = {
 
 // -------------------------------------------------------------------
 
+export function getSecurebootReadiness({ vm }) {
```

Comment:
This method should probably not be logged in the audit log: https://github.com/vatesfr/xen-orchestra/blob/0e4a300058a140341303e7c4c2a5fe16052a1df8/packages/xo-server-audit/src/index.js#L13